app/models/receiver.rb

```diff
@@ -2,4 +2,6 @@ class Receiver < ApplicationRecord
   belongs_to :user
   belongs_to :request
   has_many :comments, as: :author
+
+  enum status: { default: 0, accepted: 1, rejected: 2, commented: 3 }
```

Comment:
instead of default, i would use `pending` (which means, it waits for something: approval, reject, ...).
what do you think?